libnavui-maps/src/test/java/com/mapbox/navigation/ui/maps/internal/route/line/MapboxRouteLineUtilsTest.kt

```diff
@@ -1768,13 +1769,9 @@ class MapboxRouteLineUtilsTest {
             Point.fromLngLat(-77.167276, 38.775717),
             result.features()!![1].geometry() as Point
         )
-        assertEquals(
-            Point.fromLngLat(-77.167276, 38.775717),
-            result.features()!![2].geometry() as Point
-        )
         assertEquals(
             Point.fromLngLat(-77.153468, 38.77091),
-            result.features()!![3].geometry() as Point
+            result.features()!![2].geometry() as Point
```

Comment:
Same as above. Did the fixture change? 🤔

Reply:
Explained above.